tests/Base.php

```diff
@@ -46,38 +48,38 @@ protected static function showDeprecated(): void
     /**
      * @param array $params Additional configuration params. Host and Port are already set
      */
-    protected function _getClient(array $params = [], ?callable $callback = null, ?LoggerInterface $logger = null): Client
+    protected function _getClient(array $params = [], ?LoggerInterface $logger = null): Client
     {
-        $config = [
-            'host' => $this->_getHost(),
-            'port' => $this->_getPort(),
-        ];
+        $config = $params ?: [$this->_getHost().':'.$this->_getPort()];
 
-        $config = \array_merge($config, $params);
+        $config['transport_config']['node_pool'] = $config['transport_config']['node_pool'] ?? new TraceableSimpleNodePool(
+            new RoundRobin(),
+            new NoResurrect()
+        );
 
-        return new Client($config, $callback, $logger);
+        return new Client($config, $logger);
     }
 
     protected function _getHost(): string
     {
-        return \getenv('ES_HOST') ?: Connection::DEFAULT_HOST;
+        return \getenv('ES_HOST') ?: 'localhost';
```

Comment:
It would be nice to keep some constants but not sure where to move it. Same below.